supabase/functions/classify_feedback/index.ts

```diff
@@ -1,12 +1,123 @@
-import { serve } from "https://deno.land/std@0.131.0/http/server.ts"
+import { serve } from "https://deno.land/std@0.131.0/http/server.ts";
+import { createClient } from 'https://esm.sh/@supabase/supabase-js@^1.33.2'
+import { Configuration, OpenAIApi } from "https://esm.sh/openai@2.0.5";
 
-serve(async (req) => {
+const openai = new OpenAIApi(new Configuration({
+  organization: Deno.env.get('OPENAI_ORGANIZATION'),
+  apiKey: Deno.env.get('OPENAI_API_KEY')
+}))
+
+const supabase = createClient(
+  Deno.env.get('SUPABASE_URL') ?? '',
+  Deno.env.get('SUPABASE_ANON_KEY') ?? ''
+)
+
+const supabaseSecret = createClient(
+  Deno.env.get('SUPABASE_URL') ?? '',
+  Deno.env.get('SUPABASE_SERVICE_ROLE_KEY') ?? ''
+)
```

Comment:
Maybe should it be better to let the value by default, and let supabase throw the error if necessary? (same for others)

Reply:
It shows IDE errors, and that's what all the docs/examples do and recommend. But yes typically I would agree with you.

Reply:
Alright, if the docs/examples show it, just let's do it